streams/runners/storm/runtime/src/test/java/com/hortonworks/streamline/streams/runtime/AvroStreamsSnapshotDeserializerTest.java

```diff
@@ -66,7 +68,10 @@ public void testAvroPayloadConversions() throws Exception {
 
             new Expectations() {
                 {
-                    mockSchemaRegistryClient.getSchemaMetadataInfo(anyLong);
+                    mockSchemaRegistryClient.getSchemaVersionInfo(withEqual(SCHEMA_ID_VERSION));
```

Comment:
`withEqual` May not be needed here, actual object should be sufficient. Javadoc says

`Usually, this particular method should not be used. Instead, simply pass the desired argument value directly, without any matcher. Only when specifying values for a varargs method it's useful, and even then only when some other argument matcher is also used.`